src/Jank/TermManagement/actions.ts

```diff
@@ -0,0 +1,35 @@
+import { Term, ActionType } from './types';
+
+/**
+ * Creates and returns an action to set the list of available terms.
+ *
+ * @param terms the terms to set
+ * @returns the action that can be dispatched to set terms
+ */
+export const setTerms = (terms: Term[]) => ({
```

Comment:
Everything in this file is an action that can be dispatched to the reducer. By convention, an action consists of a `type` and a `payload` - this isn't enforced, but this is how it's usually done. The actions are very slim and are used as a means of communication between components and the reducer.